src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -830,6 +832,65 @@ public function isAbstract()
     }
 
     /**
+     * Whether this definition is deprecated, that means it should not be called
+     * anymore.
+     *
+     * @param bool   $status
+     * @param string $template Template message to use if the definition is deprecated
+     *
+     * @return Definition the current instance
+     *
+     * @throws InvalidArgumentException When the message template is invalid.
+     *
+     * @api
+     */
+    public function setDeprecated($status = true, $template = null)
+    {
+        if (null !== $template) {
+            if (preg_match('#[\r\n]|\*/#', $template)) {
```

Comment:
@Taluu : sorry for digging this up but… isn't there any way this could strip out carriage returns and trim indention instead of refusing carriage returns? This gives [very long lines](https://github.com/sonata-project/SonataAdminBundle/pull/4029/files#diff-acc840fac08a56f7459b015ba7448a9dR55)